website/docs/cli.mdx

```diff
@@ -40,7 +40,7 @@ Builds and serves a preview of your site locally with [Webpack Dev Server](https
 | `--port` | `3000` | Specifies the port of the dev server. |
 | `--host` | `localhost` | Specify a host to use. For example, if you want your server to be accessible externally, you can use `--host 0.0.0.0`. |
 | `--hot-only` | `false` | Enables Hot Module Replacement without page refresh as a fallback in case of build failures. More information [here](https://webpack.js.org/configuration/dev-server/#devserverhotonly). |
-| `--no-open` | `false` | Do not open automatically the page in the browser. |
+| `--no-open` | `false` | Do not open the page automatically in the browser. |
```

Comment:
nit: "Do not automatically open the page in the browser." sounds a bit more clear

Reply:
both look good to me, will keep current one for now